SU2_CFD/include/solvers/CScalarSolver.inl

```diff
@@ -130,8 +130,17 @@ void CScalarSolver<VariableType>::Upwind_Residual(CGeometry* geometry, CSolver**
                                                   CNumerics** numerics_container, CConfig* config,
                                                   unsigned short iMesh) {
   const bool implicit = (config->GetKind_TimeIntScheme() == EULER_IMPLICIT);
-  const bool muscl = config->GetMUSCL_Turb();
-  const bool limiter = (config->GetKind_SlopeLimit_Turb() != NO_LIMITER);
+
+  bool muscl;
+  bool limiter;
```

Comment:
Might be better to store during construction.

Reply:
07b1a8c like so?